### PR TITLE
chore(operator, yaml): add openebs operator 2.2.0

### PR DESCRIPTION
**Pull request by akhilerm**
- add openebs operator 2.2.0, RC1, RC2
- add cstor operator 2.2.0, RC1, RC2
- add ndm operator (cstor dependency) 2.2.0, RC1, RC2
- add zfs operator 1.0.1, RC1, RC2
- add arm and openebs lite operator

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

File: 2.2.0/cstor-operator-2.2.0-RC1.yaml [hunk 1]
```diff
@@ -0,0 +1,3960 @@
...
+---
+############################################################
+####                                                    ####
+####                   CStorBackup CRD                  ####
+####                                                    ####
+############################################################
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorbackups.openebs.io
```
Comment (prateekpandey14) [inline 2.2.0/cstor-operator-2.2.0-RC1.yaml:3665]: Group version has been changed to -- > `cstorbackups.cstor.openebs.io`

File: 2.2.0/cstor-operator-2.2.0-RC1.yaml [hunk 2]
```diff
@@ -0,0 +1,3960 @@
...
+spec:
+  group: openebs.io
+  version: v1alpha1
```
Comment (prateekpandey14) [inline 2.2.0/cstor-operator-2.2.0-RC1.yaml:3668]: version changes  `v1alpha1 --> v1`

**Review by prateekpandey14 — COMMENTED** [review]
Backup/Restore CRDs changes to new groupversion and v1 apis version , we can get the changes from https://github.com/openebs/cstor-operators/blob/master/deploy/crds/all_cstor_crds.yaml

File: 2.2.0/cstor-operator-2.2.0-RC1.yaml (hunk 2 above)
Comment (akhilerm) [reply 2.2.0/cstor-operator-2.2.0-RC1.yaml]: Done.

File: 2.2.0/cstor-operator-2.2.0.yaml [hunk 3]
```diff
@@ -0,0 +1,4147 @@
...
+---
+############################################################
+####                                                    ####
+####             CStorCompletedBackup CRD               ####
+####                                                    ####
+############################################################
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorcompletedbackups.openebs.io
```
Comment (mittachaitu) [inline 2.2.0/cstor-operator-2.2.0.yaml:3762]: it should be cstorcompletedbackups.cstor.openebs.io

File: 2.2.0/cstor-operator-2.2.0.yaml [hunk 4]
```diff
@@ -0,0 +1,4147 @@
...
+---
+############################################################
+####                                                    ####
+####             CStorRestore CRD                       ####
+####                                                    ####
+############################################################
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorrestores.openebs.io
```
Comment (mittachaitu) [inline 2.2.0/cstor-operator-2.2.0.yaml:3847]: here also same as above comment

Comment (akhilerm) [reply 2.2.0/cstor-operator-2.2.0.yaml:3847]: Yep. Will do that. Sorry I missed the other 2 CRDs.

File: 2.2.0/cstor-operator-2.2.0.yaml (hunk 3 above)
Comment (akhilerm) [reply 2.2.0/cstor-operator-2.2.0.yaml]: Done.

**Review by pawanpraka1 — APPROVED** [review]
zfs-operator yamls are fine.

**Review by kmova — APPROVED** [review]
openebs operators are file - general, lite and arm.

**Review by prateekpandey14 — APPROVED** [review]
lgtm : cstor-operators, ndm

**Pull request merged**